### DISCUSSION TITLE
ANDROID_YOUTUBE_CLIENT_VERSION And IOS_YOUTUBE_CLIENT_VERSION is is outdated

### DISCUSSION THREAD

**Discussion by ag2s20150909** [TeamNewPipe/NewPipe#10150] [Questions] [answered]
ANDROID_YOUTUBE_CLIENT_VERSION And IOS_YOUTUBE_CLIENT_VERSION is is outdated

**Comment by Stypox** [comment, marked as answer]
This is not a question, and also it should go in NewPipeExtractor. Feel free to open a PR there if you want.
